Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -142,40 +142,40 @@ static ByteString format_value_list(StringView name, ReadonlySpan<Value> values)
     return builder.to_byte_string();
 }
 
-ALWAYS_INLINE static ThrowCompletionOr<Value> loosely_inequals(VM& vm, Value src1, Value src2)
+ALWAYS_INLINE static ThrowCompletionOr<bool> loosely_inequals(VM& vm, Value src1, Value src2)
 {
     if (src1.tag() == src2.tag()) {
         if (src1.is_int32() || src1.is_object() || src1.is_boolean() || src1.is_nullish())
-            return Value(src1.encoded() != src2.encoded());
+            return src1.encoded() != src2.encoded();
     }
-    return Value(!TRY(is_loosely_equal(vm, src1, src2)));
+    return !TRY(is_loosely_equal(vm, src1, src2));
 }
 
-ALWAYS_INLINE static ThrowCompletionOr<Value> loosely_equals(VM& vm, Value src1, Value src2)
+ALWAYS_INLINE static ThrowCompletionOr<bool> loosely_equals(VM& vm, Value src1, Value src2)
 {
     if (src1.tag() == src2.tag()) {
         if (src1.is_int32() || src1.is_object() || src1.is_boolean() || src1.is_nullish())
-            return Value(src1.encoded() == src2.encoded());
+            return src1.encoded() == src2.encoded();
```

Comment:
We should probably also change this so that the caller doesn't need to do an exception check down this code path, but I've ran out of time right now so will have to come to check that one later

Reply:
actually there's fast paths above it so i suppose not, but anywho, i guess this is an improvement